Tooltip white-space fix (#959)
CrowdStrike/glide-core: .changeset/green-months-pay.md

---
'@crowdstrike/glide-core': patch
---

> When Tooltip is placed in a container that has `white-space: nowrap`, the `label` and `description` properly wrap.

Reviewer: ```suggestion
Tooltip's `label` and `description` now wrap even when placed in a container that has `white-space: nowrap`.
```
